src/ceph-disk/tests/ceph-disk.sh

```diff
@@ -430,52 +430,95 @@ function test_crush_device_class() {
     $ok || return 1
 }
 
+function do_test() {
```

Comment:
I wouldn't bother on ceph-disk, see https://github.com/ceph/ceph/pull/22343

Reply:
Unless that is merged, this change is still valid and saves time.